pytest/tests/mocknet/load_testing.py

```diff
@@ -0,0 +1,36 @@
+# Force each node to submit many transactions for
+# about 20 minutes. Monitor the block production time
+# stays consistent.
+
+import sys, time
+from rc import pmap
+
+from load_testing_helper import TIMEOUT
+
+sys.path.append('lib')
+import mocknet
+
+nodes = mocknet.get_nodes()
+
+print('INFO: Starting Load test.')
+
+print('INFO: Performing baseline block time measurement')
+(block_time, stdev) = mocknet.measure_average_block_time(nodes[-1])
+assert block_time < 1.01
+assert stdev < 0.1
+print('INFO: Baseline block time measurement complete')
+
+print('INFO: Setting remote python environments.')
+mocknet.setup_python_environments(nodes)
+print('INFO: Starting transaction spamming scripts.')
+mocknet.start_load_test_helpers(nodes)
+
+start_time = time.time()
+while time.time() - start_time < TIMEOUT:
+    (block_time, stdev) = mocknet.measure_average_block_time(nodes[-1])
+    assert block_time < 1.01
```

Comment:
this might be too tight. If you didn't change configuration we can use 2s instead.

Reply:
I was seeing values of 0.7s for empty blocks and 0.8s when under load from transfers, so bounding by 1 seemed safe. I'm not sure if this is expected or not. The mocknet genesis setup is in [the near-ops repo](https://github.com/nearprotocol/near-ops/blob/master/mocknet.py#L70).

Reply:
I think it's fine for now. I was worried that under load nodes will produce blocks less frequently.